.circleci/config.yml

```
@@ -0,0 +1,68 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+  
+jobs:
+  build-docs:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:7.10
+      # Specify service dependencies here if necessary
+    environment:
+      BRANCH: gh-pages 
+      REPO: git@github.com:hyphacoop/handbook.git
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run:
+          name: "Install GitBook"
+          command: |
+            npm install gitbook-cli
+
+      - run:
+          name: "Clone gh-pages"
+          command: |
+            git clone --branch $BRANCH $REPO ~/$BRANCH
+
+      - run:
+          name: "Build Site"
+          command: |
+            gitbook install && gitbook build
+            cp -r _book/* ~/$BRANCH/
```

```suggestion
            cp -R _book/* ~/$BRANCH/
```

Seems `-r` is less portable and also deprecated. New to me! (Was rather looking whether it had an expanded version, which I usually favour in scripting when no one is typing things out :) )

https://unix.stackexchange.com/a/18718/63375